src/me/StevenLawson/TotalFreedomMod/Commands/Command_admincmds

```diff
@@ -0,0 +1,22 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "A Fast way for Adminstrators to use all useful commands", usage = "/<command>")
+public class Command_admincmds extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        server.dispatchCommand(sender, "opall -c");
+        server.dispatchCommand(sender, "setl");
+        server.dispatchCommand(sender, "purgeall");
+        server.dispatchCommand(sender, "nickclean");
+        return true;
```

Comment:
@JeromSar From my perspective, when I did this, it would be sending from console, unless it was another thing but it looked alike.